pkg/component/component.go

```diff
@@ -398,6 +390,19 @@ func CreateComponent(client *occlient.Client, componentConfig config.LocalConfig
 		); err != nil {
 			return errors.Wrapf(err, "failed to create component with args %+v", createArgs)
 		}
+
+		s.End(true)
+
+		// Trigger build
```

Comment:
Curious why we are moving the below code into this section rather than keeping it within`CreateWithGit`?

Reply:
According to https://github.com/openshift/odo/issues/1971#issuecomment-521586286, build and deploy should happen outside the CreateFromGit function, so I removed it from there

Reply:
Makes sense. Thanks.